scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -124,6 +134,7 @@ case class ScalafmtConfig(
     assumeStandardLibraryStripMargin: Boolean = false,
     danglingParentheses: Boolean = false,
     poorMansTrailingCommasInConfigStyle: Boolean = false,
+    trailingCommas: TrailingCommas = TrailingCommas.preserve,
```

Comment:
I wonder what is the best default 🤔 Personally, I think `always` might be a good choice since I'd prefer scalafmt to be opinionated where possible. However, that will break code on 2.11.

I'll open a discussion to gauge opinions, until then I agree `preserve` is the safest choice 👍

Reply:
I wonder if the default could depend the dialect of the runner